rslib/backend.proto

```diff
@@ -1052,6 +1054,19 @@ message FindAndReplaceIn {
   string field_name = 6;
 }
 
+message BrowserColumns {
+  repeated BrowserColumn columns = 1;
+}
+
+message BrowserColumn {
+  string key = 1;
+  string label = 2;
+  bool is_sortable = 3;
+  bool sorts_reversed = 4;
+  bool uses_cell_font = 5;
+  bool aligns_centered = 6;
```

Comment:
"centered" maybe? Or we could make it an "alignment" enum of { START, CENTER } to allow for us possibly including end alignment in the future